### ISSUE TITLE
Motion sensor for Nest

### ISSUE THREAD

**Issue by d3bing** [KraigM/homebridge-nest#92]
First, your plugin is a life saver. 10x this than any ecobee thermostat. Nest+HomeKit FTW.
Quick question for the dev - when using an Ecobee I noticed that it gives you access to the motion sensor of the thermostat, so you can create Automation or rules when motion is detected.
Is it possible to add this to the Nest plugin? We know Nest has a motion sensor, not sure if it is accessible through APIs. If yes, that would be an awesome addition to your plugin.

**Comment by pedroserano** [comment]
@KraigM will correct me if I'm wrong but I don't see where the motion sensor is exposed in the API currently, based on Nest's list for the thermostat (the camera appears to have a motion API call)

https://developers.nest.com/documentation/api-reference

**Comment by d3bing** [comment]
darn

**Comment by dasoutherland** [comment]
@pedroserano I'm not exactly sure but I do know that the native Wink app can access the motion sensor in the nest thermostat. I use that sensor as a tigger.

**Comment by d3bing** [comment]
@dasoutherland Yes! That would be exactly what I want to do... @KraigM do you think this is possible?

**Comment by dasoutherland** [comment]
@d3bing 
It was a nice feature. I've stopped using since moving to homebridge/home app.

**Comment by jocamero** [comment]
Anyone ever find a way to expose the away or motion sensor of the Nest Thermostat or Smoke Detector?

**Comment by rambits** [comment]
Not only from the thermostat, is that possible to have the mention sensor from Nest Protect as well?
